src/android/Utilities.java

```diff
@@ -59,8 +59,10 @@ public static long getApkBuildTime(Context context) {
         Long millis;
 
         try {
+            //replace double quotes needed for correct restoration of long value from strings.xml
+            //https://github.com/Microsoft/cordova-plugin-code-push/issues/264
             millis = parseLong(context.getString(
-                context.getResources().getIdentifier("CODE_PUSH_APK_BUILD_TIME", "string", context.getPackageName())
+                context.getResources().getIdentifier("CODE_PUSH_APK_BUILD_TIME", "string", context.getPackageName().replaceAll("\"",""))
```

Comment:
If I get it right, we should replace quotes for `context.getString(...)` method result, not for `context.getPackageName().replaceAll("\"","")`